vortex-array/src/compute/binary_numeric.rs

```diff
@@ -186,13 +190,114 @@ fn arrow_numeric(
     let array = match operator {
         BinaryNumericOperator::Add => arrow_arith::numeric::add(&lhs, &rhs)?,
         BinaryNumericOperator::Sub => arrow_arith::numeric::sub(&lhs, &rhs)?,
-        BinaryNumericOperator::Div => arrow_arith::numeric::div(&lhs, &rhs)?,
+        BinaryNumericOperator::FlippedSub => arrow_arith::numeric::sub(&rhs, &lhs)?,
```

Comment:
Ok, I'm convinced enough that we should keep these. But can we call it `RSub` `RDiv` (following Python)

Reply:
Done.